internal/testserver/server.go

```diff
@@ -0,0 +1,135 @@
+package testserver
```

Comment:
> internal/testserver/server.go

I think test server could be useful for unit tests in libs. However, I'd like to avoid depending on internal/ from libs/.

Can we move this to libs/testserver?